js/controllers/routes.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+page('/about', aboutController.index);
+page('/', projectController.index);
```

Comment:
Generally the base goes above the special ones because it's easier to read, but it doesn't actually matter.